sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -1888,30 +1947,116 @@ public AzureSearchLinkedService(object url) { }
+        public object Server { get { throw null; } set { } }
+        public Azure.Analytics.Synapse.Artifacts.Models.SecretBase ServicePrincipalCredential { get { throw null; } set { } }
+        public object ServicePrincipalCredentialType { get { throw null; } set { } }
         public object ServicePrincipalId { get { throw null; } set { } }
         public Azure.Analytics.Synapse.Artifacts.Models.SecretBase ServicePrincipalKey { get { throw null; } set { } }
         public object Tenant { get { throw null; } set { } }
+        public object TrustServerCertificate { get { throw null; } set { } }
+        public object UserName { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct AzureSqlDWAuthenticationType : System.IEquatable<Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public AzureSqlDWAuthenticationType(string value) { throw null; }
+        public static Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType ServicePrincipal { get { throw null; } }
+        public static Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType SQL { get { throw null; } }
+        public static Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType SystemAssignedManagedIdentity { get { throw null; } }
+        public static Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType UserAssignedManagedIdentity { get { throw null; } }
+        public bool Equals(Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType left, Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType right) { throw null; }
+        public static implicit operator Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType (string value) { throw null; }
+        public static bool operator !=(Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType left, Azure.Analytics.Synapse.Artifacts.Models.AzureSqlDWAuthenticationType right) { throw null; }
+        public override string ToString() { throw null; }
     }
     public partial class AzureSqlDWLinkedService : Azure.Analytics.Synapse.Artifacts.Models.LinkedService
     {
-        public AzureSqlDWLinkedService(object connectionString) { }
+        public AzureSqlDWLinkedService() { }
```

Comment:
There are a lot of break changes of this pattern. Please confirm if expected add to changelog.

Reply:
Add these break changes to changelog